src/checks.rs

```diff
@@ -32,6 +32,13 @@ fn checklist() -> Vec<Box<dyn Check>> {
     ]
 }
 
+pub fn available_check_names() -> Vec<String> {
+    checklist()
```

Comment:
This method doesn't include the `EndingBlankLine` check.

Reply:
> This method doesn't include the `EndingBlankLine` check.

Yes indeed!
In this case, we still have to make a strict list of checks (as I wrote [above](https://github.com/dotenv-linter/dotenv-linter/pull/202#issue-422457368)), and when adding new ones, we will need to explicitly update the integration test

But, it seems to me, perhaps, it is conceptually more correct to add the attribute of the last line to the `LineEntry` and check directly inside the check, as in other cases.
Since we have an exception check that takes our logic out.
This, of course, is beyond the scope of this task, but this is an occasion to reflect.

What do you think about it?